src/zapv2/__init__.py

```diff
@@ -123,7 +123,7 @@ def urlopen(self, url, *args, **kwargs):
         # Must never leak the API key via proxied requests
         return requests.get(url, proxies=self.__proxies, verify=False, *args, **kwargs).text
 
-    def _request_api(self, url, query=None):
+    def _request_api(self, url, query=None, validate_status_code=False):
```

Comment:
How about allow to set this at the client level (i.e. `ZAPv2`) instead? Otherwise the user has to manually call the API endpoints(?).

Reply:
Correct, I did not thought about it...